source/guides/account/scams-and-security.rst

```diff
@@ -0,0 +1,94 @@
+
+A spoofing attack might go something like this:
+
+- You ask a question on Twitter or in the XYMCity Discord server
+- A malicious account identifies you as a target due to your request for support and sends you a DM. The account will be configured to resemble an official support resource and could include a relevant logo, a convincing username, and a professional sounding dialogue. 
+- The bad actor will rely on your belief that they are an official support resource, and try to convince you to provide your private key or mnemonic phrase key to resolve your problem. This could also involve directing you to a website that requests these details. 
+- If successful, the bad actor can gain full access to your address and transfer your assets to an address of their choosing.
+
+This scenario is just an example, and similar events could play out across any social media platform, messaging service, forum, or otherwise on which you share information publicly. 
+
+**How can I protect myself from spoofing attacks?**
+
+The Symbol Syndicate will never directly message you for customer support, and will never ask for your private key or mnemonic phrase. Anyone asking you for contact information, your secret recovery phrase or details of your support issue outside of the Discord helpdesk channel is a potential scammer and should be ignored and/or reported.
+Be vigilant. If it looks like it might be a scam, it probably is. Always be observant and keep a lookout for suspicious, telltale signs. These could include:
+
+- Requesting personal information, including anything from your name, the value of your wallet's holdings, or even your private key, which you should never, ever give to anyone. 
+- Unofficial-looking Twitter handles using underscores, doubled-up letters, and numbers to mimic official accounts. 
+- Official-looking Discord 'about me' details or handles (i.e. SUPPORT_TEAM)
+- Providing links to websites purported to 'validate your wallet' or assist with troubleshooting (excepting known-good sources such as the official Symbol documentation or articles written by Syndicate members)
+- Requests to reach out for support, get in touch, or send a DM. 
+
+Most importantly, **never share your private keys and mnemonic phrase** regardless of how convincing a person/entity may be.
+
+
+*****************
+Clipboard hacking
+*****************
+
+**What is clipboard hacking?**
+Symbol addresses are not easily memorized nor typed manually, so many users leverage their device's clipboard to copy and paste a recipient's Symbol address. Clipboard hacking involves malware that intercepts the contents of your clipboard and replaces an address you've copied with an address belonging to the hacker.  When you go to send a transaction and paste the address from your clipboard, the hacker's address is pasted instead of the one you initially copied. 
+
+**How can I protect myself?**
```

Comment:
Missing blank line after header. In the preview you can already see this is not displaying correctly. Please always check the preview :)